README.md

```diff
@@ -16,4 +16,4 @@ List of all the meetups of the OSDC Community along with the links to their resp
  2018-09-14 | [How to Open Source](https://bit.ly/2PWTA2Z)  |Addressing the know-hows of stepping into open-source.
  2018-09-28 | [Intro to Git & Github](blog-link-here)       |Introduction to version control using Git & Github.
  2019-01-23 | [Contributing to Node.js](blog-link-here)     |Getting started with contributing to Node.js.
- 
+ 2019-01-31 | [Intro to Vim](https://github.com/vaibhavk/presentations/blob/master/index.org)     |Introduction to Vim and its module. 
```

Comment:
The link must be pointing to the corresponding blog which as of now is not present.

Reply:
@darkHarry  Which blog ? There is only a index.org file on @vaibhavk presentations repo .

Reply:
@darkHarry  Someone has to the write blog . Can I ?

Reply:
yes, you can.